libs/remix-ui/static-analyser/src/lib/remix-ui-static-analyser.tsx

```diff
@@ -760,14 +733,16 @@ export const RemixUiStaticAnalyser = (props: RemixUiStaticAnalyserProps) => {
                   {tabKeys[1].child}
                 </Tab> : null
               }
-              {solhintEnabled ? <Tab
-                key={tabKeys[0].tabKey}
-                title={tabKeys[0].title}
-                eventKey={tabKeys[0].tabKey}
-                tabClassName="text-decoration-none font-weight-bold px-2"
-              >
-                {tabKeys[0].child}
-              </Tab> : null}
+              {
+                solhintEnabled ? <Tab
+                  key={tabKeys[0].tabKey}
+                  title={tabKeys[0].title}
+                  eventKey={tabKeys[0].tabKey}
+                  tabClassName="text-decoration-none font-weight-bold px-2"
+                >
+                  {tabKeys[0].child}
+                </Tab> : null
+              }
               { showSlither && slitherEnabled ? <Tab
```

Comment:
New line required to make it consistent with others.
Also here https://github.com/ethereum/remix-project/pull/3838/files#diff-a9943b741f29f3e349b333686b2558938e5ff1c853cd177eb6d8bd28e60be3a2R629

Reply:
Done